### PR TITLE
goel-vanshika updated css and color scheme of client sign in page

### PR DESCRIPTION
**Pull request by goel-vanshika**
#### Issue Number
fixes # 6

#### Describe the changes you've made
I made updates to the css to match vendor sign in page in the form of color matching, font styling and overall look of the page

#### Describe if there is any unusual behaviour of your code ( Write `NA` if there isn't )
NA

#### Additional context
NA

#### Test plan ( Optional but Recommended )
NA
#### Checklist

- [x ] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ x] The title of my pull request is a short description of the requested changes.
Screenshot 2021-09-19 at 11 57 06 AM

### CODE REVIEW

File: PRNTRS Client/main.css [hunk 1]
```diff
@@ -1,62 +1,147 @@
...
-    color:black;
-    font-family: 'Open Sans', sans-serif;
-    font-size: 22px;
-    margin-left:300px;
+.welcome
+{
+    position: absolute;
+	width: 411px;
+	height: 82px;
+	left: 237px;
```
Comment (A-HK) [inline PRNTRS Client/main.css]: Maybe just change the `left` value a bit to make sure the `Welcome Back To` comes directly above `PRNTRS CLIENT`.
Other than that, looks great!

**Review by A-HK — CHANGES_REQUESTED** [review]

**Comment by goel-vanshika** [comment]
i did make the changes.. im not sure if they are commited or not
please check once thank you

**Comment by A-HK** [comment]
Hey, nope! I don't think you've pushed your commit to this.

**Comment by A-HK** [comment]
Hey @goel-vanshika , looks great now! Please let me know when you've added the new image to the assets folder, I'll merge this PR then

**Comment by goel-vanshika** [comment]
@A-HK i have added it...and made the change to the alignment as well

**Comment by A-HK** [comment]
Hmm I still don't see the img added.
Are you sure you've pushed it here?
I just see the alignment change

**Comment by goel-vanshika** [comment]
I pushed it here again..is it working now?

**Comment by A-HK** [comment]
Yess awesome!
Will merge soon

**Pull request merged**